spec/tic_tac_toe/cli_ui_spec.clj

```diff
@@ -0,0 +1,37 @@
+(ns tic-tac-toe.cli-ui-spec
+  (:require [speclj.core :refer :all]
+            [tic-tac-toe.board-spec :refer [empty-board]]
+            [tic-tac-toe.cli-ui :refer :all]))
+
+(def delay-time 0)
+
+(describe "format-board-cli"
+          (it "formats the board for CLI"
+              (let [cli-formatted-board "1 2 3 \n4 5 6 \n7 8 9 "]
```

Comment:
When the test is this simple I wouldn't use let binding / variables - question to ask is whether the additional line of the let binding makes the test easier to read. When setup is more complex, then let binding / variables can make it more descriptive. But I don't think that's the case here.